makejdk.sh

```diff
@@ -258,14 +257,14 @@ testOpenJDKInNativeEnvironmentIfExpected()
 {
   if [[ "$JTREG" == "true" ]];
   then
-      $WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JTREG_TEST_SUBSETS
+      "$WORKING_DIR"/sbin/jtreg.sh "$WORKING_DIR" "$OPENJDK_REPO_NAME" "$BUILD_FULL_NAME" "$JTREG_TEST_SUBSETS"
```

Comment:
Should `$JTREG_TEST_SUBSETS` be splitting on spaces?  Not sure whether this is a single arg or set of args (in which case it should be an array).